games/Criminal.js

```diff
@@ -0,0 +1,260 @@
+/*
+/*
+@title: Criminal
+@author: Karen
+*/
+Instructions:
```

Comment:
```suggestion
/*
Instructions
```
Uh oh! There seems to be some issues with the comments, make sure they are properly commented.
Let us know if you need any help!